spec/requests/internal/broadcasts_spec.rb

```diff
@@ -87,4 +87,29 @@
       end
     end
   end
+
+  context "with type_of Announcement" do
+    let(:super_admin) { create(:user, :super_admin) }
+    let(:params) { { title: "Hello!", processed_html: "<pHello!</p>", type_of: "Announcement", active: true } }
```

Comment:
Same here: there is a missing `>` on the `p` tag.

Reply:
This is what happens when you copypasta pre-existing code and don't read it closely enough 🙊 Fixed now.